content/docs/command-reference/add.md

```diff
@@ -379,41 +377,24 @@ outs:
     path: data.xml
 ```
 
-## Example: Transfer to remote storage
-
-When you have a large dataset in an external location, you may want to track it
-as if it was in your project, but without downloading it locally (for now). The
-`--to-remote` option lets you do so, while storing a copy
-[remotely](/doc/command-reference/remote) so it can be
-[pulled](/doc/command-reference/plots) later.
+[linked]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+[external cache]:
+  /doc/user-guide/managing-external-data#setting-up-an-external-cache
 
-Let's add the `data.xml` to our remote storage from the given remote location:
-
-```dvc
-$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml \
-          --to-remote
-```
+## Example: Transfer to remote storage
 
-The only difference that dataset is transferred straight to remote, so DVC won't
-control the remote location you gave but rather continue managing your remote
-storage where the data is now on. The operation will still be resulted with an
-`.dvc` file:
+Use `--to-remote` to create a `.dvc` file for the operation without downloading
+data, transferring it directly to [remote storage] instead:
 
 ```dvc
-$ ls
-data.xml.dvc
+$ dvc add https://data.dvc.org/get-started/data.xml --to-remote
 ```
 
-Whenever anyone wants to actually download the added data (for example from a
-system that can handle it), they can use `dvc pull` as usual:
+See this [`import-url` example][iutr] for more details.
```

Comment:
I left the example in `add` but reduced it to its bare minimum and linked to the one in `import-url` for details.